ui-tests/cypress/integration/administration/file-browser.spec.js

```diff
@@ -1,92 +1,525 @@
-import HomePage       from '../../support/pageObjects/HomePage';
-import {htmlElements} from '../../support/pageObjects/WebElement';
+import {deleteDownloadsFolder} from '../../support/utils';
+import HomePage                from '../../support/pageObjects/HomePage';
+import {htmlElements}          from '../../support/pageObjects/WebElement';
 
-describe([Tag.GTS], 'File browser', () => {
+const apiURL = Cypress.config('restAPI');
```

Comment:
try to re-use the _intercept_ method in the _abstractController_, to remove this import; let's try to avoid the proliferation of "magic strings"

Reply:
I used the method from abstractController, though I had to edit it a little